modules/uptime-check/main.tf

```diff
@@ -84,11 +75,11 @@ metric.type="monitoring.googleapis.com/uptime_check/check_passed" AND metric.lab
       }
 
       trigger {
-        count   = 1
+        count = 1
       }
     }
   }
-  user_labels = lookup(each.value, "labels", {})
+  user_labels = var.labels
```

Comment:
so each alert will have the same labels?

Reply:
i could not find a problem with this, do you? :I

Reply:
I don't see a problem, unless we want to add a custom label to a specific alert, which is not `cc` or `clan/service name`